Makefile

```diff
@@ -213,15 +213,26 @@ test: all
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
 	$(MAKE) doc-only
+	$(MAKE) lint
```

Comment:
Non Blocking Suggestion: run `lint` before `all`

Reply:
My plan is to make `make lint` independent of the binary build, if we do that `lint` will look for a global `node` which might not exist.

Reply:
Ack. (I'm used to run it as `make lint PYTHON=python NODE=node`)

Reply:
another future TODO: `make lint-changes` that queries git for the changed files since...